plugins/staticroute/plugin_test.go

```diff
@@ -38,17 +36,42 @@ func TestSetup4(t *testing.T) {
 		assert.Equal(t, "expected a gateway address, got: foo", err.Error())
 	}
 
+	// prepare DHCPv4 request
+	req := &dhcpv4.DHCPv4{}
+	resp := &dhcpv4.DHCPv4{
+		Options: dhcpv4.Options{},
+	}
+
 	// valid route
-	_, err = setup4("10.0.0.0/8,192.168.1.1")
+	handler, err := setup4("10.0.0.0/8,192.168.1.1")
+	result, stop := handler(req, resp)
+	assert.Same(t, result, resp)
+	assert.False(t, stop)
+	table := result.Options.Get(dhcpv4.OptionStaticRoutingTable)
+	routes := dhcpv4.Routes{}
+	if err := routes.FromBytes(table); err != nil {
+		t.Errorf("FromBytes(%v) Unexpected error state: %v", table, err)
+	}
```

Comment:
This should be a different test, eg `TestHandler4`

Reply:
No problem